### DISCUSSION TITLE
Cannot install new version of Serenity.Pro.Templates

### DISCUSSION THREAD

**Discussion by klomkling** [serenity-is/Serenity#6942] [Help]
Hi everyone,

I tried to install a new template version for Startsharp with this command
```dotnet new install Serenity.Pro.Templates```
And I can install version 6.5.1 as the latest version no newer version.
So I try to install it with a specific version
```dotnet new install Serenity.Pro.Templates:6.7.1```
I got the error `Serenity.Pro.Templates:6.7.1 is not supported.`

Please help,
Thank you.

**Comment by VictorTomaili** [comment]
To access pro templates, you need to buy premium from [serenity.is](serenity.is). Thereafter, you can ask questions in StartSharp repository about pro packages. StartSharp is not a public template.
